src/frontend/sider/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -31,34 +31,91 @@ const StyledResultatKolonne = styled(Table.HeaderCell)`
     width: 22%;
 `;
 
-interface IBehandlingshistorikkProps {
-    minimalFagsak: IMinimalFagsak;
-}
+const TableHeader = () => {
+    return (
+        <Table.Header>
+            <Table.Row>
+                <StyledOpprettetKolonne children={'Opprettet'} />
+                <Table.HeaderCell children={'Årsak'} />
+                <Table.HeaderCell children={'Type'} />
+                <Table.HeaderCell children={'Behandlingstema'} />
+                <Table.HeaderCell children={'Status'} />
+                <Table.HeaderCell children={'Vedtaksdato'} />
+                <StyledResultatKolonne children={'Resultat'} />
+            </Table.Row>
+        </Table.Header>
+    );
+};
+
+type IBehandlingshistorikkProps = {
```

Comment:
Trenger vi å gi den et mer kompleks navn en `Props`? Vi prøver også å kvitte oss med `I` prefixen på interfacene (se f.eks. [her](https://dev.to/mscamargo/why-you-should-avoid-using-the-i-prefix-for-interfaces-in-typescript-43gd#:~:text=Alignment%20with%20Interface%20Segregation%20Principle,interfaces%20they%20do%20not%20use.)) så hvis du ikke liker `Props` kan du evt. vurdere `BehandlingerProps` 

```suggestion
type Props = {
```